app/views/page/show.html.erb

```diff
@@ -159,8 +159,8 @@
               elc = @event_list_components[component_index]
               @event_list_component_index = component_index + 1
             %>
-            <div class="grid-container">
-              <ul class="event-component-list usa-card-group dm-paginated-<%= component_index %>-events margin-bottom-4 <%= page_narrow_classes %><%= ' margin-bottom-0' if last_component === index %>">
+            <div class="grid-container margin-bottom-8 <%= page_narrow_classes %><%= ' margin-bottom-0' if last_component === index %>">
+              <ul class="event-component-list usa-card-group dm-paginated-<%= component_index %>-events">
                 <%= render(partial: 'events_list', locals: {events: elc[:events]}) %>
               </ul>
               <% if elc[:pagy].count > 2  %>
```

Comment:
Let's add a `margin-bottom-4` to the Load more button instead of the `<ul>`. If you only have 2 events, you'll get too much margin between elements:
![Screenshot 2023-07-24 at 11 43 37 AM](https://github.com/department-of-veterans-affairs/diffusion-marketplace/assets/5402927/f1e8522c-bbe9-4880-84c0-6a3c254a462c)

Reply:
@camillevilla not sure how the schema got in there but I just removed it from the commit, my next pr will have a few schema changes so I will rebase that one when the paperclip upgrade is merged to double check functionality :)